src/SFA.DAS.EmployerFinance.Web/Controllers/BaseController.cs

```diff
@@ -35,38 +36,23 @@ protected override ViewResult View(string viewName, string masterName, object mo
                 {
                     ModelState.AddModelError(errorMessageItem.Key, errorMessageItem.Value);
                 }
+
                 if (orchestratorResponse.Status == HttpStatusCode.BadRequest)
                 {
                     return ReturnViewResult(ControllerConstants.BadRequestViewName, masterName, orchestratorResponse);
                 }
-                return ReturnViewResult(viewName, masterName, orchestratorResponse);
-            }
-
-            if (orchestratorResponse.Status == HttpStatusCode.BadRequest)
-            {
-                return ReturnViewResult(ControllerConstants.BadRequestViewName, masterName, orchestratorResponse);
-            }
 
-            if (orchestratorResponse.Status == HttpStatusCode.NotFound)
-            {
-                return base.View(ControllerConstants.NotFoundViewName);
+                return ReturnViewResult(viewName, masterName, orchestratorResponse);
             }
 
             if (orchestratorResponse.Status == HttpStatusCode.OK)
             {
                 return ReturnViewResult(viewName, masterName, orchestratorResponse);
             }
 
-            if (orchestratorResponse.Status == HttpStatusCode.Unauthorized)
+            if (orchestratorResponse.Status >= HttpStatusCode.BadRequest)
             {
-                var accountId = Request.Params[ControllerConstants.AccountHashedIdRouteKeyName];
-
-                if (accountId != null)
-                {
-                    ViewBag.AccountId = accountId;
```

Comment:
Looks like the old access denied view might have needed the AccountId. The new one doesn't, so we don't need to supply it, but I'm wondering if we've lost something that we should be in the new view?

Reply:
Per @elliotritchie, the old view used the layout, so required the account id, whereas the new one doesn't (it's purely static), so doesn't need the AccountId.